src/ApiPort.VisualStudio/packages.config

```diff
@@ -0,0 +1,87 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should transition this project to PackageReference as well. Not needed for this review, but it'd be good to get done.